File: src/main/java/com/kfgs/statistics/service/AdministrativeAreaService.java
package com.kfgs.statistics.service;

import com.kfgs.domain.TbAdministrativeArea;
import com.kfgs.domain.response.QueryResponseResult;

import java.util.List;

/**
 * 服务层地理位置（省市县）接口
 */
public interface AdministrativeAreaService {
    /**
     * 返回 全部列表
     * @return
     */
    public QueryResponseResult findAll();

    /**
     * 根据父类ID 获取子类分类
     * @param parentId
     * @return
     */
    public QueryResponseResult findListByParentId(String parentId);

    /**
     * 获取所有包含path的地区
     * @param path
     * @return
     */
    public  List<TbAdministrativeArea> findListLikePath(String path);



}

File: src/main/webapp/js/service/administrativeAreaService.js
//统计service层
tongjiApp.service('administrativeAreaService',function ($http) {

    //获取饼状图数据
    this.selectAll= function () {
        return $http.get('../../area/selectAll.do');
    }

    this.selectByParentId = function (parentId) {
        return $http.get("../../statistics/selectByParentId.do?id="+parentId);
    }
});